substratevm/mx.substratevm/suite.py

```diff
@@ -932,6 +932,7 @@
             "subDir": "src",
             "description" : "SubstrateVM static analysis to find ahead-of-time the code",
             "dependencies": [
+                "com.oracle.svm.core",
```

Comment:
This, unfortunately, won't work as it pulls all `com.oracle.svm.core` stuff into `pointsto.jar`.